src/encoded/static/components/antibody.js

```diff
@@ -347,17 +347,20 @@ const AntibodyStatus = (props) => {
                             {Object.keys(organisms).map((organism, i) => {
                                 const terms = Object.keys(organisms[organism]);
                                 return (
-                                    <div key={i} className="antibody-status__group">
-                                        <div className="antibody-status__organism">
-                                            {organism}
+                                    <>
+                                        {i !== 0 ? <div className="antibody-status__group-gutter" /> : null }
```

Comment:
I think this could be done more easily in CSS by assigning a bottom margin to `antibody-status__group`, and then using `antibody-status__group:last-child` to have a 0 bottom margin. Saves a (granted, not very expensive) extra div, and then the only Javascript change is the `<Status>` line.

Reply:
Good catch.